pkg/install/v1alpha1/jiva_volume.go

```diff
@@ -657,13 +659,13 @@ spec:
     action: list
     options: |-
       labelSelector: openebs.io/replica=jiva-replica,openebs.io/persistent-volume={{ .Volume.owner }}
-    retry: "12,10s"
+    retry: "24,5s"
   post: |
     {{- jsonpath .JsonResult "{.items[*].metadata.name}" | trim | saveAs "createlistrep.items" .TaskResult | noop -}}
     {{- .TaskResult.createlistrep.items | empty | verifyErr "replica pod(s) not found" | saveIf "createlistrep.verifyErr" .TaskResult | noop -}}
     {{- jsonpath .JsonResult "{.items[*].spec.nodeName}" | trim | saveAs "createlistrep.nodeNames" .TaskResult | noop -}}
-    {{- $expectedRepCount := .Config.ReplicaCount.value | int -}}
-    {{- .TaskResult.createlistrep.nodeNames | default "" | splitList " " | isLen $expectedRepCount | not | verifyErr "number of replica pods does not match expected count" | saveIf "createlistrep.verifyErr" .TaskResult | noop -}}
+    {{- $expectedRepCount := div .Config.ReplicaCount.value 2 | add1 | int -}}
+    {{- .TaskResult.createlistrep.nodeNames | default "" | splitList " " | isGtorEq $expectedRepCount | not | verifyErr "number of replica pods does not match expected count" | saveIf "createlistrep.verifyErr" .TaskResult | noop -}}
```

Comment:
Can we make use of standard function i.e. `ge`?
`https://golang.org/src/text/template/funcs.go?s=1000:1035#L48`

We might need to create `splitListLen` as a new template function.
```go
func splitListLen(separator, given string) int {
  // ..
}
```

Reply:
Done